src/test_userproc.c

```diff
@@ -0,0 +1,15 @@
+int main();
+int _start() {
+    return main();
+
+    // invoke system call that kills the process
```

Comment:
When the system is properly set up, _start() should call a system call that kills the process in kernel mode. Right now we don't have it ready yet, so this is just a marker so we can come back to it in the future.